images/win/scripts/Installers/Install-PostgreSQL.ps1

```diff
@@ -0,0 +1,58 @@
+function Install-PostgreSQL {
+    $installPath= choco install postgresql | Select-String "Installed to"
+    return $installPath.Line.split("'")[1]
+}
+
+function Start-PostgreSQL {
+    param(
+        [String]$PostgresPath
+    )
+    $pgbin=Join-path $PostgresPath "bin"
+    $pgdata=Join-path $PostgresPath "data"
+    $startPostgres=Join-path $pgbin "pg_ctl.exe"
+    Start-Process -FilePath $startPostgres -ArgumentList ("-D", "$pgdata", "stop")
+    Write-Host "PostgreSQL has been successfully started."
+}
+
+function Validate-PostgreSQL {
+    param(
+        [String]$PostgresPath
+    )
+    $pgbin=Join-path $PostgresPath "bin"
+    $pgisreadypath=Join-path $pgbin "pg_isready.exe"
+    $pgready = Start-Process -FilePath $pgisreadypath -Wait -PassThru
+    $exitCode = $pgready.ExitCode
+        if ($exitCode -eq 0)
+        {
+          Write-Host -Object "PostgreSQL has been successfully installed."
+        }
+        else
+        {
+          Write-Host -Object "PostgreSQL is not ready. Exitcode: $exitCode"
+          exit $exitCode
+        }
+}
+
+function Get-PostgreSQLVersion {
+    param(
+        [String]$PostgresPath
+    )
+    $pgbin=Join-path $PostgresPath "bin"
```

Comment:
Do we need $pgbin three times?

You can temporary add to the Path binPath and remove code with Join-Path.
`$env:Path +=";$pgbin"`